profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java

```diff
@@ -246,7 +256,15 @@ public void setNeighbouringListApConfig(NeighbouringAPListConfiguration neighbou
 		this.neighbouringListApConfig = neighbouringListApConfig;
 	}
 
-	public int getMinAutoCellSize() {
+	public Integer getMinAutoCellSize() {
+		if (minAutoCellSize == null) {
```

Comment:
Maybe it's just my browser but the formatting is odd here. Otherwise code looks fine.

Reply:
The formatting looks fine on my Eclipse as well, not sure what happened there